app/services/return-requirements/check/returns-requirements.service.js

```diff
@@ -11,26 +11,26 @@ const ReturnRequirementsPresenter = require('../../../presenters/return-requirem
 /**
  * Orchestrates fetching and presenting the return requirements for `/return-requirements/{sessionId}/check` page
  *
- * @param {string} session - The session for return requirement journey
+ * @param {module:SessionModel} session - The session for the return requirement journey
  *
  * @returns {Promise<Object>} page data needed by the view template
  */
 async function go (session) {
   const { requirements, journey } = session.data
 
   const purposeIds = _purposeIds(requirements)
-  const purposes = await _fetchPurposeIds(purposeIds)
+  const purposes = await _fetchPurposes(purposeIds)
 
   return ReturnRequirementsPresenter.go(requirements, purposes, journey)
 }
 
-async function _fetchPurposeIds (purposeIds) {
+async function _fetchPurposes (purposes) {
```

Comment:
```suggestion
async function _fetchPurposes (purposeIds) {
```

Reply:
I did spot and fix this on the other branch just now !